susemanager-utils/susemanager-sls/salt-ssh/preflight.sh

```diff
@@ -202,6 +202,37 @@ elif [ "${INSTALLER}" = "apt" ]; then
     fi
 fi
 
+function selinux_enabled {
+    if command -v selinuxenabled; then
+        selinuxenabled
+    else
+        false
+    fi
+}
+
+SELINUX_POLICY_FILENAME="salt_ssh_port_forwarding.cil"
+function selinux_policy_loaded {
+    semodule -l | grep -x $SELINUX_POLICY_FILENAME
+}
+
+function write_selinux_policy {
+    if ! [[ -f $SELINUX_POLICY_FILENAME ]]; then
+        echo "(portcon tcp ${REPO_PORT} (system_u object_r ssh_port_t ((s0)(s0))))" >$SELINUX_POLICY_FILENAME
+    fi
+}
+
+function load_selinux_policy {
+    semodule -i $SELINUX_POLICY_FILENAME
+}
+
+# We need a tunnel and selinux might interfere without a custom policy
```

Comment:
Also here this is not precise. I would rather say:
Our SSH tunnel uses a custom port and we must configure SELinux to account for it

Reply:
I've included your version, thanks for the suggestion!